beacon-chain/blockchain/van_process_pending_blocks.go

```diff
@@ -8,14 +8,15 @@ import (
 	"github.com/prysmaticlabs/prysm/beacon-chain/core/feed"
 	blockfeed "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/block"
 	statefeed "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/state"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
 	"github.com/prysmaticlabs/prysm/proto/interfaces"
 	vanTypes "github.com/prysmaticlabs/prysm/shared/params"
 	"time"
```

Comment:
Unsorted imports

Reply:
I see that it is in use: `var parentPanShards []*ethpb.PandoraShard`